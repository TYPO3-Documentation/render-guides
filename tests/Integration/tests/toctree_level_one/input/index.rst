==============
Document Title
==============

Lorem Ipsum Dolor.

..  toctree::
    :maxdepth: 1
    :glob:

    *
    */index
