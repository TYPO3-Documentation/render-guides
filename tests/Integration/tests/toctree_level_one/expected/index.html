<!-- content start -->
                <section class="section" id="document-title">
    <h1>Document Title</h1>

            <p>Lorem Ipsum Dolor.</p>
            <div class="toctree-wrapper compound">
    <ul class="menu-level">
    <li class="toc-item"><a href="page1.html#page-1">Page 1</a></li>

    <li class="toc-item"><a href="subpages/index.html#subpages">Subpages</a></li>

    </ul>
</div>

    </section>

        <!-- content end -->
